src/main/java/dev/pgm/events/format/rounds/format/FormatRound.java

```diff
@@ -1,11 +1,7 @@
 package dev.pgm.events.format.rounds.format;
 
 import dev.pgm.events.format.TournamentFormat;
-import dev.pgm.events.format.rounds.AbstractRound;
-import dev.pgm.events.format.rounds.RoundDescription;
-import dev.pgm.events.format.rounds.RoundPhase;
-import dev.pgm.events.format.rounds.RoundSettings;
-import dev.pgm.events.format.rounds.TournamentRound;
+import dev.pgm.events.format.rounds.*;
```

Comment:
unsure about how this projects feels about wildcard imports, but since the diff in `VetoDescription` shows removing one maybe this is a forgotten IDE thing?

Reply:
My bad, should all be fixed now. Thanks!